monkey/infection_monkey/exploit/hadoop.py

```diff
@@ -51,7 +57,7 @@ def _exploit_host(self):
                 target=self.host.ip_addr,
                 exploitation_success=False,
                 error_message=self.exploit_result.error_message,
-                tags=(HADOOP_EXPLOITER_TAG,),
+                tags=PROPAGATION_TAGS,
```

Comment:
Why propagation tags when we publish an exploitation event?

Reply:
Good catch -- fixed!